x-pack/plugins/actions/server/builtin_action_types/lib/send_email.ts

```diff
@@ -49,12 +49,14 @@ export async function sendEmail(logger: Logger, options: SendEmailOptions): Prom
   const { from, to, cc, bcc } = routing;
   const { subject, message } = content;
 
-  const transportConfig: Record<string, any> = {
```

Comment:
This is the only real code change in the PR.  Previously, we were always setting `transportConfig.auth.user` and `pass`, even when they were null.  Now we check that they're null, and don't set them in that case.